cluster/charts/rook-ceph-cluster/templates/deployment.yaml

```
@@ -0,0 +1,65 @@
+{{- if .Values.toolbox.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: rook-ceph-tools
+  labels:
+    app: rook-ceph-tools
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: rook-ceph-tools
+  template:
+    metadata:
+      labels:
+        app: rook-ceph-tools
+    spec:
+      dnsPolicy: ClusterFirstWithHostNet
+{{- if eq .Values.cephClusterSpec.network.provider "host" }}
+      hostNetwork: true
+{{- end }}
+      containers:
+        - name: rook-ceph-tools
+          image: rook/ceph:v1.6.1
```

We need the master tag by default, then when we create the releases, we search and replace `master` with the version such as `v1.6.1`
```suggestion
          image: rook/ceph:master
```